src/utils/page.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
```

Comment:
What are `utils` in this context? These look like Next.js functions. How do we reflect that in our naming?

Reply:
Completely changed folder structure now we have:
1. `components` that holds Experiment, Variant, ABTestingContext, ExperimentContext
1. `experiments` that holds permutations
1. `next` that holds next.js interfaces for experiments (`permuteStaticPaths` and etc)
1. `webpack` that holds everything related to webpack plugin
1. `constants.ts` every constant moved to that file